index.html

```diff
@@ -2510,8 +2526,22 @@ <h5>Conclusion</h5>
 <ul data-ucr-role="conclusion-notes">
 <!-- bullet-point notes summarizing implementation details / blocking issues.
       Add tags to the start of a point with an empty `<a data-ucr-role="tag"></a>` where the text content or href matches a tag `id` value. -->
-<li><a data-ucr-role="tag" href=""></a>
-  <!-- short summary of how the tag applies -->
+<li>
+<a data-ucr-role="tag" href="#author-simplicity"></a>
+Many map services are either paid or require some form of authentication,
+even for simple use cases such as this.
+Having this capability be a browser feature
+reduces complexity for the vast majority of use cases.
+</li>
+<li>
+<a data-ucr-role="tag" href="#author-independence"></a>
+The vast majority of web maps currently require either third-party mapping libraries
+or third party tiles.
+</li>
+<li>
+<a data-ucr-role="tag" href="#accessibility-author-context"></a>
+Author input is necessary in order to provide an appropriate accessible experience,
+as any programmatic attempt to generate a textual equivalent to the map will be lacking in context.
```

Comment:
When I was writing this, I was focused on the 'for a given area' part of the capability rather than the 'default layer' part. I wanted to convey that even with a native map element, a basic map (just showing an area, no markers or anything else), requires additional context in order to be meaningful. For example, a map focused on the same area could be labelled "Map of Tri-State Area", "Map of NYC" or "Map of Manhattan Island" depending on zoom level, and it could be hard to programmatically generate meaningfully. However, if you want to keep the focus of this capability on the 'default layers' aspect, just let me know and I will remove this tag!

Reply:
The way this capability is currently described is very specific: "Generate a default map **for a given area**" - you might want to generate a default map without displaying a certain area? The "for a given area"-part looks like it should be a separate capability.

Reply:
Perhaps we need to update the language in this capability to reference 'default map layer' or 'default provider'? 'Default map' to me implies a level of functional and semantic completeness.

Reply:
Agreed on both points, I've copied them over to https://github.com/Maps4HTML/HTML-Map-Element-UseCases-Requirements/issues/8#issuecomment-539697695

If either of you wants to start a discussion page for a basic "set the initial position & scale of the map" capability, go right ahead.

With that in mind, going back to alt text: for the *default* map layer, the website author wouldn't really know the details of what is displayed, so I don't think they could improve over auto-generated description of the displayed features.